src/components/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+function Button({buttonType, buttonClass, handleClick, isDisabled, children}) {
+    return (
+        <button
+            type={buttonType}
+            disabled={isDisabled}
+            className={buttonClass}
+            onClick={handleClick}
+        >
+            {children}
+        </button>
+    );
+}
```

Comment:
Dit component voelt als een doorgeefluik. Alle props worden direct doorgegeven aan het button element. Je had dan net zo goed direct het button element kunnen gebruiken

Reply:
Ik snap wat je bedoelt. Maar in welk geval zou je dan wel een component voor je button maken? Van Elwyn begreep ik dat hij adviseerde om altijd van je buttons een component te maken, maar meestal heb je toch niet meer nodig dan de standaard eigenschappen van een button element? Als je wilt dat hij er anders uitziet zit het meestal in de css-klasse dus die kun je altijd al makkelijk meegeven.